proofs/cbmc/keccak_squeeze_once/Makefile

```diff
@@ -27,7 +27,18 @@ USE_DYNAMIC_FRAMES=1
 EXTERNAL_SAT_SOLVER=
 CBMCFLAGS=--bitwuzla
 
-CBMCFLAGS += --no-array-field-sensitivity --arrays-uf-always --slice-formula
+# For this proof we tell CBMC to
+#  1. not decompose arrays into their individual cells
+#  2. to model arrays directly as SMT-lib arrays
+#  3. to slice constraints that are not in the cone of influence of the proof obligations
+# These options simplify them modelling of arrays and produce much more compact
+# SMT files, leaving all array-type reasoning to the SMT solver.
+#
+# For functions that use large and multi-dimensional arrays, this yields
+# a substantial improvement in proof performance.
+# CBMCFLAGS += --no-array-field-sensitivity
+# CBMCFLAGS += --arrays-uf-always
```

Comment:
The comment above justifies why we should turn on all 3 flags, but then we only enable --slice-formula.
This seems a little inconsistent.

Reply:
Done